app/views/users/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <div>
 <div class="u-container">
-<h1>Displays user info</h1>
+<%# <h1>Displays user info</h1> %>
 <img class="u_image" src="<%= @user.photo ? @user.photo : 'https://images.unsplash.com/photo-1582567056798-7dc94989e56d?ixlib=rb-4.0.3&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=387&q=80' %>">
     <div class="u-info">
     <h3><%= link_to @user.name, @user %></h3>
```

Comment:
- [ ] @Dadadon Great work on your show page, however when a user has added less than 3 posts, he doesn't have a way to navigate to the posts index page, kindly rectify this by removing the if statement around your posts page link, below is how your link should look like
```
  <div>
    <%= link_to "Show All Posts", user_posts_path(@user) %>
  </div>
```

- [ ] @Dadadon Great work on the page design, however, the positioning of the posts comments and likes counters doesn't match the design given, check the comparison below

Project Design          |  Your design     
:-------------------------:|:-------------------------:
![image](https://github.com/Dadadon/blog/assets/61408860/1251ac4a-ab51-47fb-8d3a-c5b7fc81dc8c) | ![image](https://github.com/Dadadon/blog/assets/61408860/d3cce573-4e47-4d02-b641-ed3b0ae9431f)